content/realtime/channels.textile

```diff
@@ -205,6 +207,10 @@ bc[swift]. realtime.channels.get("chatroom").attach { error in
 
 Clients attach to a channel in order to participate on that channel in any way (either to publish, subscribe, or be present on the channel).
 
+h2(#channel-metadata). Channel metadata
```

Comment:
Let's discuss with Matt if we should call the other file channel metadata and not channel lifecycle as the former is more relevant